action.yml

```diff
@@ -32,17 +32,26 @@ runs:
         printf %s "${REMOTE}/${CHART_NAME}" > $CLOUDBEES_OUTPUTS/chart
         printf %s "${CHART_VERSION}" > $CLOUDBEES_OUTPUTS/version
         set -x
-        helm push "$CHART" "$REMOTE"
-        
-        # Publish build artifact
-        response=$(curl --fail-with-body  -X 'POST' "$URL/v2/workflows/runs/artifactinfos" -H "Authorization: Bearer $API_TOKEN" -H 'Content-Type: application/json' --data-binary '{"runId":"'"$RUN_ID"'","runAttempt":'$RUN_ATTEMPT',"name":"'"$CHART_NAME"'","version":"'"$CHART_VERSION"'","url":"'"$REMOTE"/"$CHART_NAME":"$CHART_VERSION"'","type":"helm"}') || command_failed=1
-        # Check curl exit code
-        if [ ${command_failed:-0} -eq 1 ];
-        then
-          echo "Platform API call failed with error '$response'"
-          exit 1
+        helm_push_output=$(helm push "$CHART" "$REMOTE" 2>&1)
```

Comment:
Was this tested for failure? If the `helm push` command fails then the script will stop execution, not giving you a chance to check for the return code. If you really want to continue on error you would need to do a `set -e` before calling the `helm push` command.

Reply:
Yes, It is verified. If the helm push is failed then it is failed

Reply:
Sure it will fail, but did you verify seeing the error messages you are logging [below](https://github.com/cloudbees-io/helm-push/blob/e2356b8a0dfe60012895b9d92ecdfd4e0b3e5a89/action.yml#L52-L54)?